torch/_inductor/memory.py

```diff
@@ -28,48 +29,35 @@ class MemoryPlanningInfoForBuffer:
     succ_nodes: OrderedSet[BaseSchedulerNode] = dataclasses.field(
         default_factory=OrderedSet
     )
-    outdegree: int = 0  # this is used only in topological_sort_lpmf
 
 
 @dataclasses.dataclass
 class MemoryPlanningInfoForNode:
```

Comment:
I am only keeping the attributes that are static. For dynamically set attributes (e.g., `memory_to_free`), those will be kept track of during the function that needs them.